x/ccv/provider/keeper/genesis.go

```diff
@@ -79,6 +79,7 @@ func (k Keeper) InitGenesis(ctx sdk.Context, genState *types.GenesisState) {
 	}
 
 	k.SetParams(ctx, genState.Params)
+	k.InitializeSlashMeter(ctx)
```

Comment:
Let's merge now, but go back and rebuild state from the genesis.

Reply:
Also export the relevant state.

Reply:
Issue for this: https://github.com/cosmos/interchain-security/issues/564